### DISCUSSION TITLE
Interactive widget does not update after join operation

### DISCUSSION THREAD

**Discussion by ankurmah** [atoti/atoti#871] [General]
I am following the main Atoti tutorial as a new user. I noticed a potential issue that I wanted to report with the interactive widget not updating with new dimensions after a join operation.

The tutorial walks users through creating a sales table, creating a cube, and creating an interactive widget. Then it walks users to reading a products table and joining the sales table with this new table. The tutorial then shows how the dimensions of the cube are automatically updated with the data from the new table and shows screenshots of how the interactive widget is updated with the new data as well (after re-running session.widget in the Jupyter Lab). However, I do not see the widget updated with the data from the new table.

Am I doing something incorrectly, or are users expected to regenerate the cube? If it's the latter, the tutorial needs to be updated to reflect that.

**Comment by andrewyang94** [comment]
Hey @ankurmah thanks for posting about this! Just to make sure we're on the same page, could you perhaps add a few screenshots (or even better a video) for when you're facing this issue? This way we can see exactly what code you're running, and what issues you're referencing. 

To be clear, when doing a join in the cube, the data model and underlying hierarchies should be automatically refreshed (i.e. the left-hand pane for the JupyterLab extension), but not necessarily the data itself (you will have to re-run the `session.widget`). If this is not the case, then we'll have to diagnose further. Looking forward to hearing from you!

Also ccing my colleague @tibdex
